tasks/basic-sanity-check.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: basic-sanity-check
```

Comment:
Should we rename the task and the file to sanity-label-check? Since we plan on adding separate tasks for sanity-deprecated-image check and sanity-signed-rpm-check, etc. ?